templates/documents/create.html.twig

```diff
@@ -2,52 +2,46 @@
 
 {% block title %}{% trans %}Documents{% endtrans %}{% endblock %}
 
-{% set backlink = "/case/" ~ case.id ~ "/documents" %}
-
 {% block header %}
     <div class="row">
         <div class="col">
-            <h1>{% trans %}Create new document{% endtrans %}</h1>
+            <h1>{% trans %}Upload document{% endtrans %}</h1>
         </div>
         <div class="col-auto">
-            <a name="" id="" class="btn btn-secondary" href={{ backlink }} role="button">{% trans %}Cancel{% endtrans %}</a>
-            <a name="" id="" class="btn btn-success" href="#" role="button">{% trans %}Save changes{% endtrans %}</a>
+            <a name="" id="" class="btn btn-secondary" href="{{ url('case_documents', {id: case.id}) }}" role="button">{% trans %}Cancel{% endtrans %}</a>
         </div>
     </div>
 {% endblock %}
 
 {% block content %}
 
-    <main role="main" class="col-12 mt-3">
+    {{ form_start(document_form) }}
```

Comment:
Will this not produce another `<form>` element?

Reply:
Yes. Avoided duplicate `<form>` elements.